bin/load_db_scxa_cell_clusters.sh

```diff
@@ -79,7 +79,7 @@ if [ -n "$CONDENSED_SDRF_TSV" ]; then
   for additionalCellGroupType in "${additionalCellGroupTypes[@]}"; do
     grep -m 1 "$(printf '\t')$additionalCellGroupType$(printf '\t')" $CONDENSED_SDRF_TSV >/dev/null
     if [ $? -eq 0 ]; then
-        grep "$(printf '\t')$additionalCellGroupType$(printf '\t')" $CONDENSED_SDRF_TSV | awk -F'\t' 'BEGIN { OFS = "|"; } {print $1,$3,$5,$6}' >> $groupMembershipsToLoad    
+        grep "$(printf '\t')$additionalCellGroupType$(printf '\t')" $CONDENSED_SDRF_TSV | sort | uniq | awk -F'\t' 'BEGIN { OFS = "|"; } {print $1,$3,$5,$6}' | sed 's/ *$//' >> $groupMembershipsToLoad    
```

Comment:
much more cleverer!

Reply:
Only if it's actually identical - we shall see!